doc/contribute_models.md

```diff
@@ -110,36 +149,37 @@ Write an SQLFlow statement to test the model using iris dataset (you need to imp
 ```sql
 %%sqlflow
 SELECT * FROM iris.train
-TO TRAIN my_awesome_model.MyAwesomeClassifier
+TO TRAIN MyAwesomeClassifier
 WITH model.n_classes=3
 LABEL class
 INTO models_db.awesome_model;
 ```
 
-you may go back to `mydnnclassifier.py` and modify the model code until it works as you expected.
+you may go back to `some_model_definitions.py` and modify the model code until it works as you expected.
 
 ## Publish Your Model
 
 In the final step, you need to publish your model so that other SQLFlow users can get the model and use it.
 
-1. If you are contributing to https://github.com/sql-machine-learning/models, file a pull request on Github to merge your code to SQLFlow's models repo. The model should be available when SQLFlow's Docker image `sqlflow/sqlflow` is updated.
+1. If you are contributing to https://github.com/sql-machine-learning/models, file a pull request on Github to merge your code to SQLFlow's models repo.  The model should be available when SQLFlow's Docker image `sqlflow/sqlflow` is updated.
```

Comment:
There's a redundant whitespace before the second sentence.

Reply:
See: https://github.com/sql-machine-learning/sqlflow/pull/2132#discussion_r414173828